packages/SymbioticController/composer.json

```diff
@@ -4,7 +4,8 @@
         "php": "^7.1",
         "nette/di": "^2.4",
         "nette/application": "^2.4",
-        "odbav-to/presenter-route": "^0.1"
+        "odbav-to/presenter-route": "^0.1",
```

Comment:
I mean `PresenterRoute` class.

It implements already `IRoute` from Nette. The constructor is what relevant, not the implementation.

> To be honest it does not fit in here for me (the whole PresenterRoute package).

This is and always will optional, since it have to be used manually in Router of the application.